### PR TITLE
Tag out IRC & Google+ and tag in discord!

### PR DESCRIPTION
**Pull request by clone1018**
Let's resurge the open company initiative! The first step, getting anyone who's interested together in a central place. 

I created a Discord server for the Open Company Initative:
Screen Shot 2020-03-04 at 5 38 41 AM

### CODE REVIEW

**Comment by galuszkak** [comment]
@clone1018  I never used discord. Is this similar to Slack?

**Comment by clone1018** [comment]
It's very similar, the main reason I am pitching it over slack is because of my experience with a local not-for-profit programming Slack. We have so many users chatting so frequently we run out of messages, and much history & knowledge is lost.

Screen Shot 2020-03-04 at 5 54 53 AM

Discord is simple, lightweight, has everlasting history and we can easily drop in a bot to make chat logs public (planned).

File: _i18n/en/join/index.md [hunk 1]
```diff
@@ -30,3 +30,9 @@ with the following information:
 ### Dues
 
 Membership dues are pay what you want via [Liberapay](https://liberapay.com/opencompany/).
+
+
+### Chat
+
+We maintain an [Discord Server](https://discord.gg/MDffbaQ) where our members, or any interested parties can join to discuss open companies, the initative, or their specific initatives.
```
Comment (chadwhitacre) [inline _i18n/en/join/index.md]:
```suggestion
We maintain a [Discord Server](https://discord.gg/MDffbaQ) where our members, or any interested parties can join to discuss open companies, the initiative, or their specific initiatives. 
```

**Comment by galuszkak** [comment]
@waldyrious do you have thoughts on this? 
I think I'm in favor if this has to help us to move this initiative again.

**Comment by galuszkak** [comment]
As there isn't any response from @waldyrious I'm merging this in.

**Comment by galuszkak** [comment]
@clone1018 merged! Should be deployed in next 2-5 min.

**Pull request merged**